### PR TITLE
feat(typst): add raw block injections

### PR DESCRIPTION
**Pull request by LamprosPitsillos**

### CODE REVIEW

File: queries/typst/injections.scm [hunk 1]
```diff
@@ -1,2 +1,7 @@
 ((comment) @injection.content
   (#set! injection.language "comment"))
+
+(raw_blck
+  (ident) @_lang
+  (blob) @injection.content
+  (#set-lang-from-info-string! @_lang))
```
Comment (clason) [inline queries/typst/injections.scm:7]: This is deprecated due to upstream changes.

Comment (LamprosPitsillos) [reply queries/typst/injections.scm:7]: Oh sorry, i just looked at what markdown query did.

Comment (LamprosPitsillos) [reply queries/typst/injections.scm:7]: i checked some other `injections.scm`  and dont see a way *other than manually writing it for every language*
 
 Am i missing something?

Comment (clason) [reply queries/typst/injections.scm:7]: `@injection.language`

Comment (lucario387) [reply queries/typst/injections.scm:7]: Yes, yes you do
https://github.com/neovim/neovim/blob/bd5008de07d29a6457ddc7fe13f9f85c9c4619d2/runtime/lua/vim/treesitter/languagetree.lua#L759

Comment (lucario387) [reply queries/typst/injections.scm:7]: Also you can always set it through `vim.treesitter.language.register, which we do set in `parsers.lua`

Comment (clason) [inline queries/typst/injections.scm:7]:
```suggestion
(raw_blck
  (ident) @injection.language
  (blob) @injection.content
```

**Comment by clason** [comment]
@RaafatTurki

File: queries/typst/injections.scm [hunk 2]
```diff
@@ -1,2 +1,6 @@
 ((comment) @injection.content
   (#set! injection.language "comment"))
+
+(raw_blck
+  (ident) @injection.language
+  (blob) @injection.content
```
Comment (lucario387) [inline queries/typst/injections.scm:6]: Missing a closing paren here

Comment (lucario387) [inline queries/typst/injections.scm:6]: Missing a paren here

Comment (clason) [reply queries/typst/injections.scm:6]: Yeah, my bad.

Comment (clason) [reply queries/typst/injections.scm:6]: Wrong place ;) We enforce a query format now (you can run `./scripts/format-queries.lua queries/typst`).

Comment (LamprosPitsillos) [reply queries/typst/injections.scm]: Thanks for the help!

Comment (clason) [reply queries/typst/injections.scm]: (and please squash)

**Review by lucario387 — APPROVED** [review]

**Comment by amaanq** [comment]
squashed this time :sweat_smile:

**Pull request merged**